doc/reference/bluetooth/mesh/shell.rst

```diff
@@ -539,6 +616,14 @@ The Configuration Client uses the general messages parameters set by ``mesh dst`
 	* ``Company ID``: If present, determines the Company ID of the model. If omitted, the model is a Bluetooth SIG defined model.
 
 
+``mesh krp <NetKeyIdx> [Transition]``
+-------------------------------------
+
+	Get or set the krp value.
+
+	* ``NetKeyIdx``: The identified network key used to Get/Set the current Key Refresh Phase state.
+	* ``Transition``: New Key Refresh Phase Transition.
```

Comment:
`phase` instead?

What are the valid values for this?

Reply:
4.3.2.59 Config Key Refresh Phase Set

Reply:
Yes, perhaps it could be listed in the docs?

Reply:
How to do ?

Reply:
```suggestion
	* ``Phase``: New Key Refresh Phase. Valid phases are 0, 1 or 2.
```